src/elements/common/with-router-and-ref/__tests__/withRouterAndRef.test.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from 'enzyme';
+import withRouterAndRef from '../withRouterAndRef';
+
+describe('elements/common/with-router-and-ref/withRouterAndRef', () => {
```

Comment:
Could this be `elements/common/routing` or something to try to avoid folder sprawl?